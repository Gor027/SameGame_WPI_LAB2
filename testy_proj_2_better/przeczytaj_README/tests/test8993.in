 $$III$    $$ I$  $$I IIII$I$I$I$ I$ $$$I   $III       $ I 
 I$$ $ $I  $  IIII$  I I$$ $$$$I$ $$II I  $$ $
I$  II$$  III$$$I$  $ $$$I I II $ $$   $ I$  II
I $   II I$  I $I$I $II I$  $I $ $I$II$
I  I  
$  $I $II   I$  $ I  II  $ I$$$
$III $$ $ I$$I$  I  I$ II$I  $$I  $$I$ $$IIIIII I $I$ I $III$$I$I
$$II  $$ $  I I$I $II $I I III $$$ I$I  $$$$$  $$$I$I
$I$II$I$I I I$
I$   II III $$$I   I I$I
I$$$I$II 
 I$$I$I I$$$I  $$I  II$$$$$$ $I$ $  III$  
I$$IIIIII$    $I  II$  II $$$ II$I$I$
