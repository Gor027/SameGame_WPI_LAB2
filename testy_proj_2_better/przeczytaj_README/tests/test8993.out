      $
   I  $
 $$$ II
 I$I $$$    $  $       I I   I
I$$$II$$I   I II    I III$ I $$I   $
IIIII$$II I I$I$I $ $I$$$I $$I$I  I$I$
I$III$II$I$$$I$$II$ $III$$ I$II$$ $$II $
$I$IIII$I$I$I$$$I$I$IIIII$$I$$$I$$$$II I  $  I
$$$$I$I$IIIIIIII$$III$$II$II$$$$$$I$$$$$II$$$$I
$$$$I$III$I$$$$$II$III$$$I$$III$$$$III$$$IIIIIIII $I$   I
$$$IIIIII$$$I$$I$III$I$II$$$$$II$I$I$II$$$$$$II$$$I$II$$III$$I$I
